gradio/components/logout_button.py

```diff
@@ -37,6 +38,9 @@ def __init__(
         scale: int | None = 0,
         min_width: int | None = None,
     ):
+        warnings.warn(
```

Comment:
I would also add a note on line 17 mentioning that `LogoutButton` is deprecated and to use `LoginButton` instead so that this information shows up in our docs